integreat_cms/nominatim_api/nominatim_api_client.py

```diff
@@ -242,47 +243,29 @@ def get_region_bounding_box(self, name, aliases=None):
             bounding_boxes.append(BoundingBox.from_result(self.search(city=alias)))
         return BoundingBox.merge(*bounding_boxes)
 
-    def reverse(
-        self, query_str=None, exactly_one=True, addressdetails=False, **query_dict
-    ):
-        r"""
-        Search for a given query, either by string or by dict.
-        ``query_str`` and ``query_dict`` are mutually exclusive.
-
-        :raises RuntimeError: When the ``query_str`` and ``query_dict`` parameters are passed at the same time
-
-        :param query_str: The query string
-        :type query_str: str
-
-        :param exactly_one: Whether only one result should be returned
-        :type exactly_one: bool
+    def get_address(self, latitude, longitude):
+        """
+        Get coordinates for given address
 
-        :param addressdetails: Whether address details should be returned
-        :type addressdetails: bool
+        :param latitude: The requested latitude
+        :type latitude: str
 
-        :param \**query_dict: The query as dictionary
-        :type \**query_dict: dict
+        :param longitude: The requested longitude
+        :type longitude: str
 
-        :return: The location that matches the given criteria
-        :rtype: geopy.location.Location
+        :return: The address at these coordinates
+        :rtype: dict
         """
-        if query_str and query_dict:
-            raise RuntimeError(
-                "You can either specify query_str or pass additional keyword arguments, not both."
-            )
-
-        query = query_str or query_dict
```

Comment:
This does not make sense in this case because the [`reverse()` function](https://geopy.readthedocs.io/en/stable/#geopy.geocoders.Nominatim.reverse) of the underlying library only accepts a `query` parameter and no dictionary.